README*.md

```diff
@@ -1,3 +1,8 @@
+\* = `plotlyupdate.m` is currently disabled. Please download the latest version 
```

Comment:
Yeah - that's intentional so that old versions of the wrapper 404 on update. It's really the only to shutoff the update. We could leave it as is and just let the old wrappers fail to update when they hit the error described in #124, but this way is more intentional/cleaner.